OpenRA.Game/Game.cs

```diff
@@ -392,6 +393,18 @@ public static void InitializeMod(string mod, Arguments args)
 
 			JoinLocal();
 
+			try
+			{
+				if (Game.Settings.Server.DiscoverNatDevices)
```

Comment:
Minor nit: it would be more correct and robust to check `if (discoverNat != null)`.